lib/rules/selector-list-comma-newline-after/index.js

```diff
@@ -45,20 +45,22 @@ const rule = function(expectation) {
           functionArguments: "skip"
         },
         match => {
-          const nextThreeChars = selector.substr(match.endIndex, 3);
+          const nextChars = selector.substr(
+            match.endIndex,
+            selector.length - match.endIndex
+          );
 
           // If there's a // comment, that means there has to be a newline
           // ending the comment so we're fine
-          if (nextThreeChars === " //") {
+          if (nextChars.match(/^\s+\/\//)) {
             return;
           }
 
           // If there is a space and then a comment begins, look for the newline
```

Comment:
Can this comment be updated to reflect the new functionality of the code please

How about: `If there are spaces and then a comment begins, look for the newline`